### PR TITLE
Fix allowRoutingThroughFriendRooms resolves #34

### LINKED ISSUE #34: [Reminder] AllowRoutingThroughFriendlyRooms does not work.
Not sure why its not working its set to true but its still avoiding the room. If I commend out the Occupied check it works, so its related to the check itself, or my friendsfile ?!

friends.js
module.exports =['abc','defg'];

### PR DESCRIPTION
**Pull request by Somotaw**
Fixing this typo, maybe this will resolve it? Testing plz :)

### CODE REVIEW

**Comment by Somotaw** [comment]
I have added console messages for testing

**Comment by TooAngel** [comment]
nice catch, looks good

**Comment by Somotaw** [comment]
Testing seems ok, ready for merge?

File: src/prototype_creep_routing.js [hunk 1]
```diff
@@ -23,9 +23,11 @@ Creep.prototype.getRoute = function() {
 
       if (Memory.rooms[roomName] && Memory.rooms[roomName].state == 'Occupied') {
         //         console.log(`Creep.prototype.getRoute: Do not route through occupied rooms ${roomName}`);
-        if (config.allowRoutingThroughFriendRooms && friends.indexOf(Memory.rooms[roomName].player) > -1) {
+        if (config.path.allowRoutingThroughFriendRooms && friends.indexOf(Memory.rooms[roomName].player) > -1) {
+          console.log('routing through friendly room' + roomName);
           return 1;
         }
+        console.log('Not routing through enemy room' + roomName);
```
Comment (TooAngel) [inline src/prototype_creep_routing.js]: I would remove the log output. For debugging the other should be enough. This one increases the noise.

**Comment by Somotaw** [comment]
fixed :)

**Comment by Somotaw** [comment]
hehe just removing my project file.... ;)

**Pull request merged**